### PR TITLE
Add slim.js@2.6.1

### PR DESCRIPTION
**Pull request by eavichay**
Pull request for issue: #10985
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/eavichay/slim.js
 * Official website (optional, not the repository): http://slimjs.com
 * NPM package url (optional): slim-js
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 8
   - Count of stars: 131
   - Count of forks: 5
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [V] I'm the author of this library
   * [V] I would like to add link to the page of this library on CDNJS on website / readme
 * [V] This lib was not found on cdnjs repo
 * [V] No already exist / duplicated issue and PR
 * [V] The lib has notable popularity
   * [V] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [V] More than 500 downloads stats per month on npm registry
 * [V] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [V] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [V] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [V] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
@eavichay would you please give us more details about this library? As you can see that there is a template described what kind of info we need, thanks.

**Comment by eavichay** [comment]
just committed package.json

**Comment by PeterDaveHello** [comment]
@eavichay mind to take a look at the CI error?

**Comment by PeterDaveHello** [comment]
@eavichay mind to take a look at the CI error?

File: ajax/libs/slim.js/package.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  "filename": "slim.js",
+  "version": "2.6.1",
+  "description": "Superpower Web Components",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/eavichay/slim.js.git"
+  },
+  "author": "Avichay Eyal <eavichay@gmail.com>",
+  "license": "MIT",
+  "homepage": "https://github.com/eavichay/slim-web-components#readme",
```
Comment (sufuf3) [inline ajax/libs/slim.js/package.json]: I think you would like to use http://slimjs.com/ as the homepage.
Then, you can use `"homepage": "http://slimjs.com/"` .

File: ajax/libs/slim.js/package.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "slim.js",
+  "filename": "slim.js",
```
Comment (sufuf3) [inline ajax/libs/slim.js/package.json]: Would you like to provide the minified `.js` file?
If not, it's fine. 
CDNJS would provide it with [uglify-js](https://www.npmjs.com/package/uglify-js), so please modify to "filename": "Slim.min.js".

**Comment by PeterDaveHello** [comment]
Moved to #11015

**Pull request closed without merging**